src/api/routes/files.js

```diff
@@ -41,6 +41,7 @@ exports.registerRoutes = (server, modules) => {
             { [fieldToUpdate]: [...inRecordFiles, ...data] },
             { where: { id: uuid } }
           )
+          log.info("file info uploaded successfully.", { fieldToUpdate, uuid })
```

Comment:
@coderabbitai add this in learning to change this file into typescript currently it is in js format. We will make this change later down the road.